component/alert/alert.css

```diff
@@ -0,0 +1,34 @@
+.alert {
+    padding:0.75rem 1.25rem;
+    border-radius: 0.25rem;
+    margin-bottom: 1rem;
+    border: 1px solid transparent;
+    max-height: 2rem;
+}
+.alert-primary {
+    background-color: #7fe7ff;
+    color:#1c3940;
+    border-color: #78dfd7;
+}
+.alert-danger {
+    background-color: #ff232387;
+    color:#6a0f0feb;
+    border-color: #d3939de8;
+}
+
+.alert-warning {
+    background-color: #ffb72387;
+    color: #6a0f0feb;
+    border-color: #d3939de8;
+}
+
+.alert-success {
+    background-color: #59eca5d4;
```

Comment:
use css variables for colors , put them in main css file

Reply:
Thanks mohammed. Making the changes.